pkg/externalauthprovider/flags.go

```diff
@@ -324,6 +324,21 @@ func CreateExternalAuthConfig(args *ExternalAuthProvidersArgs) (*cmv1.ExternalAu
 	externalAuthBuilder := cmv1.NewExternalAuth().ID(args.name)
 	claimValidationRules := args.claimValidationRule
 
+	// check parameters
+	if args.name == "" {
+		return &cmv1.ExternalAuth{}, fmt.Errorf(
+			"name parameter is mandatory for creating an external authentication configuration")
+	}
+
+	if args.issuerUrl == "" {
+		return &cmv1.ExternalAuth{}, fmt.Errorf(
+			"issuer-url parameter is mandatory for creating an external authentication configuration")
+	}
+	if args.issuerAudiences == nil {
+		return &cmv1.ExternalAuth{}, fmt.Errorf(
+			"issuer-audience parameter is mandatory for creating an external authentication configuration")
+	}
+
 	if args.issuerUrl != "" || args.issuerCaFile != "" || args.issuerAudiences != nil {
```

Comment:
I wonder if we can reduce this now:
```if args.issuerUrl != "" || args.issuerCaFile != "" || args.issuerAudiences != nil {```
Now that we return when `args.issuerUrl == ""` and when `args.issuerAudiences == nil` , it feels redundant to have them in this. Maybe  we can just have `if args.issuerCaFile != "" { ... }`

Reply:
WDYT?